packages/nc-gui/components/smartsheet/header/VirtualCell.vue

```diff
@@ -153,11 +153,11 @@ const openDropDown = (e: Event) => {
   >
     <LazySmartsheetHeaderVirtualCellIcon v-if="column && !props.hideIcon" />
 
-    <a-tooltip placement="bottom">
+    <a-tooltip placement="bottom" :title="column.title">
       <template v-if="!isForm && !isExpandedForm" #title>
         {{ tooltipMsg }}
       </template>
-      <span class="name truncate pl-1" :class="{ truncate: !isForm }" :data-test-id="column.title">
+      <span class="name pl-1" :class="{ truncate: !isForm }" :data-test-id="column.title">
```

Comment:
Is removing truncation for form intentional?

Reply:
**fixed in below commit**

[fix : fixed tooltip for virtual cell](https://github.com/nocodb/nocodb/pull/7114/commits/e84c4684d660cf2acfffa9b86e223b4c63480c21)
[e84c468](https://github.com/nocodb/nocodb/pull/7114/commits/e84c4684d660cf2acfffa9b86e223b4c63480c21)